test/generated/testproto/comment_special_chars_pb2.pyi

```diff
@@ -17,20 +17,36 @@ class Test(google.protobuf.message.Message):
     C_FIELD_NUMBER: builtins.int
     D_FIELD_NUMBER: builtins.int
     E_FIELD_NUMBER: builtins.int
+    F_FIELD_NUMBER: builtins.int
+    G_FIELD_NUMBER: builtins.int
+    H_FIELD_NUMBER: builtins.int
+    I_FIELD_NUMBER: builtins.int
     a: typing.Text = ...
-    """Ending with " """
+    r"""Ending with " """
 
     b: typing.Text = ...
-    """Ending with "" """
+    r"""Ending with "" """
 
     c: typing.Text = ...
-    """Ending with \"\"\" """
+    r"""Ending with \"\"\" """
 
     d: typing.Text = ...
-    """Ending with \\"""
+    r"""Ending with \ """
 
     e: typing.Text = ...
-    """Containing bad escape: \\x"""
+    r"""Containing bad escape: \x"""
+
+    f: typing.Text = ...
+    r"""Containing \"\"\"" quadruple"""
+
+    g: typing.Text = ...
+    r"""Containing \"\"\""" quintuple"""
```

Comment:
this actually seems to work ok!

Reply:
But these are not docstrings, they are somewhat ignored by the python compiler and IDEs as if they were comments ... as such I think they should remain as comments, and only actual docstrings should be made into strings.

I see that you test for comments like:
```
// this comment has "quotes"
// this comment has """Triple quotes"
```

Have you covered the case where we would have this in the proto files:
```
// "quotes" can be a problem in comments.
// """Triple quotes""" just as well
```

Reply:
These are PEP257 Attribute Docstrings - defined as:

String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called "attribute docstrings".

Reply:
I can add these two tests. I have seen comments like this before.
Good idea to add them as test cases.

You're right that testing these scenarios are unfortunately tricky.
Also totally agree that docstrings are probably a mis-feature. I am a bigger fan of comments being comments - with docutils-esque programs parsing comments rather than relying on strings / `__doc__` attributes (rust does this for example). That being said - my fanhood is not relevant - gotta do what works for the python ecosystem.

Reply:
Ah, I was not aware of Attribute Docstrings. You learn something new every day.

Reply:
I learned about the distinction from you in fact - when you linked the PEP! Thanks for educating us about it.
I knew I had tested it out with PyCharm - but hadn't dug into the weeds about what it was looking for.